config/alphagov.yml

```diff
@@ -387,6 +387,25 @@ govwifi:
     - robinmitra
     - sarahseewhy
 
+  include_repos:
+    - govwifi-acceptance-tests
+    - govwifi-admin
+    - govwifi-authentication-api
+    - govwifi-build
+    - govwifi-concourse-deploy-pipeline
+    - govwifi-concourse-runner
+    - govwifi-dashboard
+    - govwifi-dev-docs
+    - govwifi-frontend
+    - govwifi-logging-api
+    - govwifi-product-page
+    - govwifi-safe-restarter
+    - govwifi-shared-frontend
+    - govwifi-tech-docs
+    - govwifi-terraform
+    - govwifi-user-signup-api
+    - govwifi-watchdog
+
```

Comment:
I believe these are missing from the list:

* govwifi-builder-task
* govwifi-database-backup
* govwifi-redirect

Reply:
Thanks Robin.  I wasn't aware these existed; I've now amended the commit.